README.md

```diff
@@ -1,134 +1,64 @@
-# Flask React Project
+# Oney
+[Oney](https://shoponey.herokuapp.com/) is a shopping web-app for handmade products. You can check out the web site by going to the link below.
 
-This is the starter for the Flask React project.
+<div align="center">👉👉👉 <a href="https://shoponey.herokuapp.com/">Live Website Deployed Here</div>
 
-## Getting started
+## Screenshots of App in Action
 
-1. Clone this repository (only this branch)
+❌Will add screen shots once we are done❌
 
-   ```bash
-   git clone https://github.com/appacademy-starters/python-project-starter.git
-   ```
+## Summary of main features
 
-2. Install dependencies
+[Oney](https://shoponey.herokuapp.com/) includes MVP functionality for the following features:
+- Product listing
+		- Customers can add, edit, and delete their own products to the shop
+		- Customers can browse other users' product listings
+- Product reviews
+	- Customers can leave product review
+	- Customers can edit or delete their own reviews
+	- Customers can read reviews left by other users
+-  Product / Category browser
+	- All products belong to one of 253 categories and sub-categories
+	- Customers can browse products within specific categories using the category browser
+- ❌ Shopping car // WORK IN PROGRESS ❌
```

Comment:
Minor typo with the cart, but we'll be updating it once all the details are in place

Reply:
For sure!